pointcloud/pointcloud_utils.go

```diff
@@ -94,14 +94,14 @@ func StatisticalOutlierFilter(meanK int, stdDevThresh float64) (func(PointCloud)
 }
 
 // ToBasicOctree takes a pointcloud object and converts it into a basic octree.
-func ToBasicOctree(cloud PointCloud) (*BasicOctree, error) {
+func ToBasicOctree(cloud PointCloud, confidenceThreshold int) (*BasicOctree, error) {
 	if basicOctree, ok := cloud.(*BasicOctree); ok {
 		return basicOctree, nil
```

Comment:
If the passed confidenceThreshold does not match that of the underlying octree, we should skip over this and construct a new one below as if it were any other pointcloud

Reply:
I think my most recent commit addresses this but also a little confused on what youre asking

Reply:
Yes it does